docs/user-guide/src/irso/introduction.md

```diff
@@ -74,12 +74,13 @@ A major and minor version can be supplied to the `Ironic` resource to request
 a specific branch of ironic-image (and thus Ironic). Here are supported version
 values for each branch and release of the operator:
 
-| Operator version | Ironic version(s)        | Default version |
-| ---------------- | ------------------------ | --------------- |
-| latest (main)    | latest, 29.0, 28.0, 27.0 | latest          |
-| 0.3.0            | latest, 29.0, 28.0, 27.0 | latest          |
-| 0.2.0            | latest, 28.0, 27.0       | latest          |
-| 0.1.0            | latest, 27.0             | latest          |
+| Operator version | Ironic version(s)              | Default version |
+| ---------------- | ------------------------------ | --------------- |
+| latest (main)    | latest, 30.0, 29.0, 28.0, 27.0 | latest          |
+| 0.4.0            | 30.0, 29.0, 28.0, 27.0         | 30.0            |
+| 0.3.0            | latest, 29.0, 28.0, 27.0       | latest          |
+| 0.2.0            | latest, 28.0, 27.0             | latest          |
+| 0.1.0            | latest, 27.0                   | latest          |
 
 **NOTE:** the special version value `latest` always installs the latest
 available version of ironic-image and Ironic.
```

Comment:
Like, update this note with "This version value is supported by all releases of IrSO but only works reliably in the `latest` release."